public/keymaps/t/themadnoodle_noodlepad_v1_default.json

```diff
@@ -10,9 +10,14 @@
       "KC_CALC", "KC_MAIL",       "LT(1,KC_MYCM)"
     ],
     [
-      "RGB_MOD", "RGB_TOG", "RGB_RMOD",
-      "RGB_SPI", "RGB_SPD", "RGB_VAI",
-      "RGB_SAI", "RGB_SAD", "KC_TRNS"
+      "RGB_MOD",        "RGB_TOG", "RGB_RMOD",
+      "RGB_VAD",        "RGB_M_P", "RGB_VAI",
+      "LT(3,RGB_M_SW)", "RGB_M_B", "KC_TRNS"
```

Comment:
This LT won't work.

Reply:
I can swap it, but someone should take that up with the keyboard maintainer.

https://github.com/qmk/qmk_firmware/blob/4e369d405af6bba1adce6337b2e1b1ea1788566c/keyboards/themadnoodle/noodlepad/keymaps/default/keymap.c#L38